presto-hive/src/main/java/io/prestosql/plugin/hive/orc/OrcReaderConfig.java

```diff
@@ -16,22 +16,20 @@
 import io.airlift.configuration.Config;
 import io.airlift.configuration.ConfigDescription;
 import io.airlift.units.DataSize;
+import io.prestosql.orc.OrcReaderOptions;
 
 import javax.validation.constraints.NotNull;
 
-import static io.airlift.units.DataSize.Unit.MEGABYTE;
-
 public class OrcReaderConfig
 {
     private boolean useColumnNames;
-    private boolean bloomFiltersEnabled;
 
-    private DataSize maxMergeDistance = new DataSize(1, MEGABYTE);
-    private DataSize maxBufferSize = new DataSize(8, MEGABYTE);
-    private DataSize tinyStripeThreshold = new DataSize(8, MEGABYTE);
-    private DataSize streamBufferSize = new DataSize(8, MEGABYTE);
-    private DataSize maxBlockSize = new DataSize(16, MEGABYTE);
-    private boolean lazyReadSmallRanges = true;
+    private OrcReaderOptions options = new OrcReaderOptions();
```

Comment:
This is clever, but I think it's the wrong approach. We expect the options class to always be valid, but that isn't true for config classes due to how bean validation works.